code/modules/mob/dead/new_player/preferences_setup.dm

```diff
@@ -6,6 +6,8 @@
 	real_name = pref_species.random_name(gender,1)
 	if(gender_override)
 		gender = gender_override
+	//else if(pref_species.name == "Seelie")	//Male seelie allowed for now, leaving this for future reference
+	//	gender = FEMALE
```

Comment:
Remove this, and shouldn't this be an istype check on a datum(I really wish we had enums :/)

Reply:
At this point I dont believe the Seelie mob existed in the context to check that, instead we checked player preferences to disallow the option to be anything but Female in their prefs. pre_species was the in-context way of doing this, but its kind of a moot point now since Male Seelie are supported; Ill remove the commented lines